Examples/.gitignore

```diff
@@ -0,0 +1 @@
+mapbox_access_token
```

Comment:
I guess the recommended way is to use `~/.mapbox`
Ref: https://docs.mapbox.com/help/troubleshooting/private-access-token-android-and-ios/

Reply:
I can address that in PR https://github.com/mapbox/mapbox-search-ios/pull/144 / SSDK-523